src/magic_folder/config.py

```diff
@@ -849,7 +842,7 @@ def get_local_snapshot(self, cursor, name):
         return _construct_local_snapshot(
             leaf_identifier,
             name,
-            self.author,
+            self._get_author.__wrapped__(self, cursor),
```

Comment:
`__wrapped__` doesn't appear to be a documented feature of `functools.wraps`.  I'm only guessing that's what `six.wraps` actually resolves to...

I suspect you could move the `with_cursor` decorator back on to `author` and then have `_get_author` available as the function you want to have here, but I'm not sure.  If not, I guess it's fine ... if someone messes with `__wrapped__` then I guess this code will break and the tests will tell us, not a big deal...

Reply:
`__wrapped__` was added in python 3.2, as something set by `update_wrapper` (which `wraps` is documented as calling).